### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by talanknight**
Ran the following to get updated deps from go-dbw and go-kms-wrapping:
```
go get github.com/hashicorp/go-dbw@7ed943176e5b456cc796212da52af4cc9de2dbe3
go get github.com/hashicorp/go-kms-wrapping/v2@91d44aa8e36096df16f02f20737c0e5e6e168c95
go mod tidy
```

### CODE REVIEW

**Review by johanbrandhorst — APPROVED** [review]

File: internal/auth/oidc/rewrapping_test.go [hunk 1]
```diff
@@ -25,10 +25,10 @@ func TestRewrap_authMethodRewrapFn(t *testing.T) {
 		conn, mock := db.TestSetupWithMock(t)
 		wrapper := db.TestWrapper(t)
 		mock.ExpectQuery(
-			`SELECT \* FROM "kms_schema_version" WHERE 1=1 ORDER BY "kms_schema_version"\."version" LIMIT 1`,
+			`SELECT \* FROM "kms_schema_version" WHERE 1=1 ORDER BY "kms_schema_version"\."version" LIMIT \$1`,
```
Comment (johanbrandhorst) [inline internal/auth/oidc/rewrapping_test.go:28]: Why was this change necessary?

**Review by johanbrandhorst — APPROVED** [review]

File: internal/auth/oidc/rewrapping_test.go (hunk 1 above)
Comment (jimlambrt) [reply internal/auth/oidc/rewrapping_test.go:28]: I believe this was the upstream change: https://github.com/go-gorm/postgres/commit/dc711bd7c69394788e6532ad9c89f5bdcb12b007

Comment (johanbrandhorst) [reply internal/auth/oidc/rewrapping_test.go:28]: Thanks!
